system_metrics_collector/test/system_metrics_collector/test_constants.hpp

```diff
@@ -27,23 +27,21 @@
 namespace test_constants
 {
 constexpr const std::chrono::milliseconds kMeasurePeriod{50};
-constexpr const std::chrono::milliseconds kPublishPeriod{80};
+constexpr const std::chrono::milliseconds kPublishPeriod_old{80};
```

Comment:
Can you add a comment as to why this is still required?

Reply:
Sure. It will be removed in #103